lean4-infoview/src/infoview/index.css

```diff
@@ -23,13 +23,17 @@ html,body {
 .goal-vdash { color: #569cd6; }
 .goal-case { color: #a1df90; }
 .goal-hyp { color: #ffcc00; }
-.goal-inaccessible { color: #808080 }
+.goal-inaccessible {
+    color: #808080;
+    font-style: italic;
+    font-weight: normal;
+}
 
 .vscode-light .goal-goals { color: #367cb6; }
 .vscode-light .goal-vdash { color: #367cb6; }
 .vscode-light .goal-case { color: #1f7a1f; }
 .vscode-light .goal-hyp { color: #cc7a00; }
-.vscode-light .goal-inaccessible { color: #777; }
+.vscode-light .goal-inaccessible { color: #aaa; }
```

Comment:
This color choice was intentional because otherwise it's hard to read.  https://github.com/leanprover/vscode-lean4/commit/921151063ba40bda31490011829245fb33139737